cypress/integration/imprint_spec.js

```diff
@@ -0,0 +1,26 @@
+context("Imprint page", function () {
+  describe("common infos on all imprint pages", function() {
+    [
+      "/de/imprint",
+      "/imprint"
+    ].forEach(url => {
+      it("contains the GPG key on " + url, function() {
+        cy.visit(url)
+        cy.contains("2F71FB99").should("exist")
```

Comment:
Oops, I'll need to update my key ID...
```suggestion
        cy.contains("E5FF731E").should("exist")
```